styles.css

```diff
@@ -1,3 +1,793 @@
+    line-height: 26px;
+    color: #C2C3C5;
+}
+
+.info-detail {
+    font-family: "Bakbak One", Arial, sans-serif;
+    font-size: 24px;
+    font-weight: 400;
+    line-height: 30px;
+}
+
+.caption-date,
+.caption-author {
+    font-family: "Bakbak One", Arial, sans-serif;
+    font-size: 16px;
+    font-weight: 400;
+    line-height: 24px;
+    color: #888B8E;
+}
+
+.article-tag {
+    font-family: "Bakbak One", Arial, sans-serif;
+    font-size: 14px;
+    font-weight: 400;
+    line-height: 22px;
+}
+
+.blog-details>.article-author,
+.article-date {
+    font-family: Poppins, Arial, sans-serif;
+    font-size: 20px;
+    font-weight: 400;
+    line-height: 30px;
+    color: #888B8E;
+}
+
+.quote {
+    font-family: Poppins, Arial, sans-serif;
+    font-size: 24px;
+    font-weight: 600;
+    line-height: 30px;
+}
+
+.rating {
+    font-family: Poppins, Arial, sans-serif;
+    font-size: 14px;
+    font-weight: 400;
+    line-height: 22px;
+    color: #BBBABE;
+}
+
+.blog-details>.article {
+    font-family: "Bakbak One", Arial, sans-serif;
+    font-size: 30px;
+    font-weight: 400;
+    line-height: 42px;
+}
+
+.tag-item a,
+.sidebar-article p,
+.sidebar-article time,
+.tags-list li a {
+    font-family: Poppins, Arial, sans-serif;
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+}
+
+.article-text {
+    font-family: Poppins, Arial, sans-serif;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 22px;
+    color: #888B8E;
+}
+
+.article-figure .sub-text,
+.sidebar-article>h3 {
+    font-family: Poppins, Arial, sans-serif;
+    font-size: 14px;
+    font-size: 18px;
+    font-weight: 700;
+    line-height: 28px;
+    color: #fff;
+}
+
+.categories-link, .filter-item {
+    font-family: Poppins, Arial, sans-serif;
+    font-size: 14px;
+    font-weight: 400;
+    line-height: 22px;
+    color: #C2C3C5;
+
+}
+
+h2[class~="sidebar"] {
+    font-family: "Bakbak One", Arial, sans-serif;
+    font-size: 24px;
+    font-weight: 400;
+    line-height: 30px;
+}
+
+.email-form {
+    background-color: #C2C3C526;
+    width: 690px;
+    height: 56px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    border: none;
+
+}
+
+.catalog {
+    width: 441px;
+}
+
+.email-input {
+    background-color: transparent;
+    border: none;
+    color: white;
+    height: 56px;
+    width: 600px;
+    padding-left: 15px;
+    border: none;
+}
+.details-input,
+.details-textarea,
+.details-reply-textarea {
+    background-color: #C2C3C526;
+    border: none;
+    height: 54px;
+    padding: 14px 24px;
+    margin: 10px 0;
+    color: #fff;
+    font-family: Poppins, Arial, sans-serif;
+    font-size: 16px;
+}
+.details-form {
+    display: flex;
+    flex-direction: column;
+    padding: 0 615px;
+    margin: auto;
+}
+.input-group {
+    display: flex;
+}
+.leave-reply {
+    text-align: left;
+    padding: 0 646px 0 254px;
+}
+.input-reply-group {
+    display: flex;
+}
+input[class~="name"] {
```

Comment:
Style inputs by their full class, not by attribute, because inputs may differ on different pages. Imagine your colleague adds an input with the class `pokemon-name` on a new page and unexpectedly inherits the styles you set here.

Using attribute selectors for inputs can lead to unintended side effects.
If another developer adds a different input class that includes the same string, they might unintentionally inherit styles meant for something else. It’s safer and more predictable to style inputs using their full class names, ensuring that only the intended elements are affected. This approach reduces the risk of conflicts and makes your code easier to maintain.